cli/list.go

```diff
@@ -22,6 +22,7 @@ type workspaceListRow struct {
 	codersdk.Workspace `table:"-"`
 
 	// For table format:
+	Favorite       bool   `json:"-" table:"favorite"`
```

Comment:
It feels a bit weird to occupy this much prime real-estate at the beginning of the table for this, but I think I understand the motivation for having it first (to make the sort order obvious).

Would it be weird to have favorite be "Fav" and `*`/empty instead of `true/false` 🤔. Or would it be feasible to have it towards the end of the table?

Reply:
Yeah I was playing around with that idea as well. 
Another alternative I was considering was to append an asterisk `(*)` to `workspace.Name` to indicate favorite status.

Reply:
I thought about that too, and I like it, but I worry if it might be problematic (re: parsing), then again, anyone parsing should just be using `--output=json`. So maybe try that out?

Reply:
Yeah that's a fair point.
Aside: it might be good to check if `stdout` is a pipe and warn to `stderr` to use JSON format instead for parsing. Similar to what the `apt` command does:

```
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
```

Reply:
> Another alternative I was considering was to append an asterisk (*) 

I think we are also using emojis to highlight things, for example "Workspace is outdated <hand>". Maybe we could just have a "star" for favorite ones?

Reply:
> > Another alternative I was considering was to append an asterisk (*)
> 
> I think we are also using emojis to highlight things, for example "Workspace is outdated ". Maybe we could just have a "star" for favorite ones?

Something like this?

```
WORKSPACE      TEMPLATE  STATUS   HEALTHY  LAST BUILT  CURRENT VERSION   OUTDATED  STARTS AT  STOPS AFTER  
admin/test     docker    Stopped           16d17h      peaceful_yonath2  false                1d           
★ admin/test2  docker    Failed            3d22h       peaceful_yonath2  false                1d           
```

Reply:
or ⭐️ + column padding

Reply:
Padding to the width of an emoji glyph is going to be tricky; AFAICT there doesn't appear to be a space character equal to the width of an emoji.